lib/media/ver10/media.ex

```diff
@@ -18,7 +17,7 @@ defmodule Onvif.Media.Ver10.Media do
     content = generate_content(operation, args)
     soap_action = operation.soap_action()
 
-    (uri <> @endpoint)
+    uri
```

Comment:
I think we should just change the function to accept a Device struct and then build the uri here based on host, username, password, and media_path. Otherwise every request module is gonna have to do that and it would be less work to do it in one place